app/themes/dashboard/ui/styles/theme.less

```diff
@@ -38,4 +38,9 @@ z-index: 10;
 
 .main-header #logo {
    text-indent: 99999px;
-}
\ No newline at end of file
+}
+
+.pre-xml {
+  // background-color: white;
```

Comment:
@jenbreese removed. How does the collapsed sidebar look? Do you want a small MarkLogic image or ML logo in a similar style?